lighthouse-core/lib/lh-error.js

```diff
@@ -24,7 +53,9 @@ class LighthouseError extends Error {
     super(errorDefinition.code);
     this.name = 'LHError';
     this.code = errorDefinition.code;
-    this.friendlyMessage = errorDefinition.message;
+    this.friendlyMessage = (errorDefinition.substitute) ?
```

Comment:
WDYT about having good names for each substitution and just pass the whole errorDefinition in as second arg?